ports/stm32f4/boards/espruino_wifi/mpconfigboard.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Lucian Copeland for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+//Micropython setup
+
+#define MICROPY_HW_BOARD_NAME       "Espruino Wifi"
+#define MICROPY_HW_MCU_NAME         "STM32F411xE"
+
+#define FLASH_SIZE                  (0x80000) //512K
+#define FLASH_PAGE_SIZE             (0x4000)  //16K
+
+#define BOARD_OSC_DIV 8
+#define BOARD_OVERWRITE_SWD
```

Comment:
Please do:
```suggestion
#define BOARD_OVERWRITE_SWD (1)
```

and then #if when used. That way the compiler can verify it is available. (Also, add a default into mpconfigport.h)

Reply:
Do you want me to do this to all the other ones I've been using?
```
#define BOARD_NO_VBUS_SENSE
#define BOARD_VTOR_DEFER
#define BOARD_USE_INTERNAL_SPI
```
etc. Or alternatively, I could move them all to mpconfigboard.mk like `MCU_SUB_VARIANT` so everything is in the same place. I haven't been putting in defaults because I've been basing basically everything off the peripherals pins.h file, which doesn't use a default:
```
#ifdef NRF52840
#include "nrf52840/pins.h"
#endif
```

Reply:
Also, do you mean that you want me to `#undef BOARD_OVERWRITE_SWD` so I can redefine it? I've never seen that syntax in Circuitpython before.

Reply:
Ah, doing those others would be great. Sorry I didn't think of this sooner. By default I mean something like:
```
#ifndef foo
#define foo (0)
#endif
````